bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/thw_5f_ckm_LO_ctcvcp/thw_5f_ckm_LO_ctcvcp_run_card.dat

```diff
@@ -40,8 +54,8 @@
 #*********************************************************************
 # Renormalization and factorization scales                           *
 #*********************************************************************
-  True	=  fixed_ren_scale   ! if .true. use fixed ren scale
-  True	=  fixed_fac_scale   ! if .true. use fixed fac scale
+  False	=  fixed_ren_scale   ! if .true. use fixed ren scale
+  False	=  fixed_fac_scale   ! if .true. use fixed fac scale
```

Comment:
do you want the default setup, cuz you are using "-1	=  dynamical_scale_choice"

Reply:
I would like to have the default setup, isn't `-1 = dynamical_scale_choice` the default? 
It is also chosen in the tHq run card.

Reply:
-1 is the default setup